Tests/Adyen Tests/Analytics/AnalyticsProviderTests.swift

```diff
@@ -34,18 +35,18 @@ class AnalyticsProviderTests: XCTestCase {
         let checkoutAttemptIdResponse = CheckoutAttemptIdResponse(identifier: expectedCheckoutAttemptId)
         let checkoutAttemptIdResult: Result<Response, Error> = .success(checkoutAttemptIdResponse)
         apiClient.mockedResults = [checkoutAttemptIdResult]
-
+        
         let fetchCheckoutAttemptIdExpection = expectation(description: "checkoutAttemptId completion")
 
         // When
-        sut.fetchCheckoutAttemptId { receivedCheckoutAttemptId in
-
-            // Then
-            XCTAssertNotNil(receivedCheckoutAttemptId, "The checkoutAttemptId is nil.")
-            XCTAssertEqual(expectedCheckoutAttemptId, receivedCheckoutAttemptId, "The received checkoutAttemptId is not the expected one.")
-            fetchCheckoutAttemptIdExpection.fulfill()
-        }
-
+        sut.sendInitialAnalytics(with: .components(type: .achDirectDebit), additionalFields: nil)
+        fetchCheckoutAttemptIdExpection.fulfill()
```

Comment:
Looks like this expectation is not really doing anything anymore - We could get rid of it alltogether with the `waitForExpectations`